lms/services/canvas_api/client.py

```diff
@@ -61,6 +62,7 @@ def __init__(
         authenticated_client,
         file_service: FileService,
         pages_client: CanvasPagesClient,
+        application_instance: ApplicationInstance,
```

Comment:
We are going to explicitly DI the application instance to be able to log its ID.